columnflow/calibration/cms/jets.py

```diff
@@ -841,11 +843,21 @@ def jer_setup(self: Calibrator, reqs: dict, inputs: dict, reader_targets: Insert
         for name, key in jer_keys.items()
     }
 
+    # use deterministic seeds for random smearing if requested
+    if self.use_deterministic_seeds:
+        bit_generator = np.random.SFC64
+        def deterministic_normal(loc, scale, seed):
+            return np.asarray([
+                np.random.Generator(bit_generator(_seed)).normal(_loc, _scale)
```

Comment:
```suggestion
    if self.deterministic_seed_index >= 0:
        idx = self.deterministic_seed_index
        bit_generator = np.random.SFC64
        def deterministic_normal(loc, scale, seed):
            return np.asarray([
                np.random.Generator(bit_generator(_seed)).normal(_loc, _scale, size=idx + 1)[-1]
```

Reply:
I think there is a typo in this suggestion - the names of the input arguments does not match the name of the variables used in L852 @riga

Reply:
> I think there is a typo in this suggestion - the names of the input arguments does not match the name of the variables used in L852 @riga

Yeah its a little hard to see, but this is intended.  I use the variables immediately in a list comprehensions, where the variables are then used with an _.